### PR TITLE
Add a simple example for Spartan 6

### PR DESCRIPTION
**Pull request by koriakin**

### CODE REVIEW

File: examples/mimas2/run_yosys.ys [hunk 1]
```diff
@@ -0,0 +1,4 @@
+read_verilog example.v
+synth_xilinx -top example -family xc6s
+iopadmap -bits -outpad OBUF I:O -inpad IBUF O:I
+write_edif -pvector bra example.ndf
```
Comment (eddiehung) [inline examples/mimas2/run_yosys.ys]: Why `.ndf` file extension and not `.edif`?

Comment (koriakin) [reply examples/mimas2/run_yosys.ys]: Good point, fixed.

File: examples/mimas2/run_yosys.ys [hunk 2]
```diff
@@ -0,0 +1,4 @@
+read_verilog example.v
+synth_xilinx -top example -family xc6s
+iopadmap -bits -outpad OBUF I:O -inpad IBUF O:I
```
Comment (eddiehung) [inline examples/mimas2/run_yosys.ys]: Perhaps I was too hasty to merge it in. Would it be useful to have this `iopadmap` command be part of `synth_xilinx -family xc6s`?

Comment (koriakin) [reply examples/mimas2/run_yosys.ys]: To be discussed in #1235

**Pull request merged**